docs/modules/ROOT/pages/vshn-managed/keycloak/delete.adoc

```diff
@@ -14,3 +14,44 @@ $ oc delete vshnkeycloak.vshn.appcat.vshn.io keycloak-app1-prod
 ====
 If you delete the `VSHNKeycloak` object from the cluster without having xref:vshn-managed/postgresql/deletion-protection.adoc[deletion-protection] enabled, the Keycloak instance will be deleted together with all of its data and backups!
 ====
+
+== Delete protected Keycloak instance
+
+A Keycloak instance that is protected from deletion cannot be deleted right away.
+First disable deletion protection then delete the instance.
+
+. Disable deletion protection:
++
+[source,bash]
+----
+$ oc edit vshnkeycloak.vshn.appcat.vshn.io keycloak-app1-prod
+----
++
+[source,yaml]
+----
+apiVersion: vshn.appcat.vshn.io/v1
+kind: VSHNKeycloak
+metadata:
+  name: keycloak-app1-prod
+  namespace: prod-app
+spec:
+  parameters:
+    security:
+      deletionProtection: false # <1>
+  writeConnectionSecretToRef:
+    name: keycloak-creds
+----
+<1> Make sure the parameter is set to false
+
+. Delete the instance
++
+[source,bash]
+----
+$ oc delete vshnkeycloak.vshn.appcat.vshn.io keycloak-app1-prod
+----
++
+[IMPORTANT]
+.Non-reversible operation
+====
+If you delete the `VSHNKeycloak` object from the cluster, it will be deleted together with all of its data!
```

Comment:
And the backups!

You could probably add this to the reference as well and then do an include.

Reply:
I've moved this part to the reference and included the backup part as well